test/functional/example_y2038.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test covering the Year 2038 problem.
+
+https://en.wikipedia.org/wiki/Year_2038_problem
+"""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import (
+    assert_equal,
+)
+
+EPOCH_2038_BEFORE = 2147440306 # (GMT): Monday, January 18, 2038 3:11:46 PM
+EPOCH_2038_AFTER = 2147613106 # (GMT): Wednesday, January 20, 2038 3:11:46 PM
+EPOCH_2106 = 4293404400000
```

Comment:
This is AD 138022, not AD 2038.

Reply:
You are correct. Thank you for pointing this out. 

```
ƛ date -r 4293404400000 '+%m/%d/%Y'
07/27/138022
```

I'm going to change it to: 4293404400 to test for 2106.  See the last sentence on: https://en.bitcoin.it/wiki/Block_timestamp 

```
date -r 4293404400 '+%m/%d/%Y'
01/19/2106
```